lib/request-wrapper.ts

```diff
@@ -156,11 +156,15 @@ export class RequestWrapper {
    * @returns the string representation of the request
    */
   private formatAxiosRequest(request: InternalAxiosRequestConfig): string {
-    const { method, url, data, headers } = request;
-
+    const { method, url, data, headers, params } = request;
```

Comment:
This change just adds the query parameters to the log output, which came in handy while verifying that the authenticator is working!